test/check_sharding_metadata_test.lua

```diff
@@ -219,3 +297,132 @@ function g.test_invalid_key_reference_path()
     )
     t.assert_equals(ok, nil)
 end
+
+function g.test_sharding_func_invalid_name()
+    g.space.sharding_func = 'bad_sharding_func'
```

Comment:
Nit: The name is valid, but the function is not exists.

Reply:
Fixed.